GsaGH/Components/2_Geometry/EditNode.cs

```diff
@@ -59,7 +59,7 @@ public override bool Read(GH_IReader reader) {
         Params.ReplaceInputParameter(new GsaSpringPropertyParameter(), 7, true);
         Params.ReplaceOutputParameter(new GsaSpringPropertyParameter(), 7);
       }
-
+      Params.UpdateRestrainedBool6Parameter();
```

Comment:
is this for upgrading existing components with Bool6Params?

Reply:
Yes....No release are set by ReleaseParameter and restrained by RestraintParameter. So, It is essential to update